modules/ossm-accessing-bookinfo-application-using-istio-gateway-injection.adoc

```diff
@@ -0,0 +1,57 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-accessing-bookinfo-application-using-istio-gateway-injection"]
+= Accessing the Bookinfo application by using Istio gateway injection
+:context: ossm-accessing-bookinfo-application-using-istio-gateway-injection
+
+Gateway injection uses the same mechanisms as {istio} sidecar injection to create a gateway from a `Deployment` resource that is paired with a `Service` resource. The `Service` resource can be made accessible from outside an {ocp-product-title} cluster.
+
+.Prerequisties
+
+* You are logged in to the {ocp-product-title} web console as `cluster-admin`.
+
+* The {SMProductName} Operator must be installed.
+
+* The {istio} resource must be deployed. 
+
+.Procedure
+
+. Create the `istio-ingressgateway` deployment and service by running the following command at the CLI:
++
+[source,terminal]
+----
+$ oc apply -n bookinfo -f ingress-gateway.yaml
+----
++
+[NOTE]
+====
+This example uses a sample `ingress-gateway.yaml` https://github.com/istio-ecosystem/sail-operator/blob/eb874d1605c0401887c30d96312c6db1006d89dd/chart/samples/ingress-gateway.yaml[file] that is available in the Istio community repository. 
```

Comment:
Considering this is product documentation, do we want to include also HorizontalPodAutoscaler and PodDisruptionBudget like we do in 2.x [doc](https://docs.openshift.com/container-platform/4.15/service_mesh/v2x/ossm-traffic-manage.html#ossm-deploying-automatic-gateway-injection_traffic-management)?

Reply:
@FilipB I've added the two steps from the 2.x doc. I wasn't sure if they should be the last two steps, but I added them after the step that applies gateway configuration and before the step to ping the server and confirm things are up and running. I can move them if that makes more sense. I guess I'm thinking of pinging to confirm as the last thing someone does, but maybe not?

Reply:
Thank you Filip, I see you approved this PR. I appreciate your help!